api/envoy/admin/v2alpha/config_dump.proto

```diff
@@ -111,6 +118,9 @@ message ClustersConfigDump {
   // Envoy configuration from a configuration dump, the warming clusters should generally be
   // discarded.
   repeated DynamicCluster dynamic_warming_clusters = 4 [(gogoproto.nullable) = false];
+
+  // The timestamp when the ClustersConfig was last updated.
+  google.protobuf.Timestamp last_updated = 5;
```

Comment:
With incremental xDS, each individual resource will have its own last updated timestamp. Thoughts on whether we should just go ahead and do this now (and have the user infer the max last updated from the set)?

Reply:
That sounds good to me. I could add it to each of DynamicListener, DynamicCluster, DynamicRouteConfig. Is there interest in wrapping the Bootstrap proto in a similar struct containing version / update time?

Reply:
I don't think bootstrap needs special treatment here, so what you suggest sounds good.

Reply:
I like this, but I want to distinguish that field as `instantiation_time` or something like that, which is different in meaning (and greppability) from `last_updated`, which maybe exists at higher levels.

Reply:
I added more granular timestamps to dynamic Listeners/Clusters/Routes.